cloudanalysis/get_fv3sar_bk_parallel_mod.f90

```diff
@@ -778,8 +806,167 @@ subroutine update_fv3sar(mype)
 
   deallocate(d3r4)
 
-  call general_sub2grid_destroy_info(s)
+!  call general_sub2grid_destroy_info(s)
```

Comment:
Added condition:
if(.not.l_cld_uncertainty) call general_sub2grid_destroy_info(s)

Reply:
Thanks for the suggestion, I added the condition and reran the test. Everything looks okay now.